src/main/java/ladder/domain/Location.java

```diff
@@ -0,0 +1,47 @@
+package ladder.domain;
+
+import java.util.Objects;
+
+public class Location implements Comparable<Location> {
+    public static final Location ZERO = new Location(0);
+    private final int location;
```

Comment:
```suggestion
    public static final Location ZERO = new Location(0);
    
    private final int location;
```
상수와 필드는 새행으로 분리해두면 더 깔끔해 보입니다